tests/data/mmd-to-iso.xsl

```diff
@@ -0,0 +1,479 @@
+                <xsl:element name="gmd:northBoundLatitude">
+                    <xsl:element name="gco:Decimal">
+                        <xsl:value-of select="mmd:north" />
+                    </xsl:element>
+                </xsl:element>
+            </xsl:element>
+        </xsl:element>                
+
+    </xsl:template>
+    
+    <xsl:template match="mmd:geographic_extent/mmd:polygon">
+    
+        <xsl:element name="gmd:geographicElement">
+            <xsl:element name="gmd:EX_BoundingPolygon">
+                <xsl:element name="gmd:polygon">
+                    <xsl:copy-of select="gml:Polygon" />
+                </xsl:element>
+            </xsl:element>
+        </xsl:element>                
+
+    </xsl:template>    
+
+    <xsl:template match="mmd:data_access">
+    
+        <xsl:element name="gmd:onLine">
+            <xsl:element name="gmd:CI_OnlineResource">
+                <xsl:element name="gmd:linkage">
+                    <xsl:element name="gmd:URL">
+                        <xsl:variable name="myprot" select="normalize-space(./mmd:type)" />
+                        <xsl:choose>
+                            <xsl:when test="contains($myprot,'OGC WMS')">
+                                <xsl:variable name="myurl" select="normalize-space(./mmd:resource)" />
+                                <xsl:choose>
+                                    <xsl:when test="contains($myurl,'?SERVICE=WMS&amp;REQUEST=GetCapabilities')">
+                                        <xsl:value-of select="mmd:resource" />
+                                    </xsl:when>
+                                    <xsl:otherwise>
+                                        <xsl:value-of select="concat($myurl,'?SERVICE=WMS&amp;REQUEST=GetCapabilities')" />
+                                    </xsl:otherwise>
+                                </xsl:choose>
+                            </xsl:when>
+                            <xsl:otherwise>
+                                <xsl:value-of select="mmd:resource" />
+                            </xsl:otherwise>
+                        </xsl:choose>
+                    </xsl:element>
+                </xsl:element>
+                <xsl:element name="gmd:protocol">
+                    <xsl:element name="gco:CharacterString">
+                        <!--xsl:value-of select="mmd:type" / -->
+                        <xsl:variable name="mmd_da_type" select="normalize-space(./mmd:type)" />
+                        <xsl:variable name="mmd_da_mapping" select="document('')/*/mapping:data_access_type[@mmd=$mmd_da_type]/@iso" />
+                        <xsl:value-of select="$mmd_da_mapping" />
+                    </xsl:element>
+                </xsl:element>            
+                <xsl:element name="gmd:name">
+                    <xsl:element name="gco:CharacterString">
+                        <xsl:value-of select="mmd:name" />
+                    </xsl:element>
+                </xsl:element>
```

Comment:
there is an issue for pycsw which is addressed with a new version of this file see: https://github.com/metno/mmd/pull/106

Reply:
I see that this PR is still evolving. I'll get the new XSL file once it's settled.

Reply:
There will be two XSL files, the one already on the mmd repository ([`mmd-to-iso.xsl`](https://github.com/metno/mmd/blob/master/xslt/mmd-to-iso.xsl)) and a new one `mmd-to-pycsw.xsl` which will import from it and apply the changes needed by `pycsw` (discussed in: https://github.com/metno/mmd/pull/106)